package.json

```diff
@@ -26,7 +26,9 @@
     "build": "next build",
     "start": "rm -r .next && next build && next start",
     "lint": "eslint 'src/**/*.{js,ts,tsx}'",
-    "prettier:format": "yarn prettier './**/*.{js,json,md,ts,tsx,yml}' --write && yarn run lint --fix"
+    "prettier:format": "yarn prettier './**/*.{js,json,md,ts,tsx,yml}' --write && yarn run lint --fix",
+    "e2e:open": "env-cmd -f .e2e.env yarn synpress open --configFile synpress.config.ts",
+    "e2e:run": "env-cmd -f .e2e.env yarn synpress run --configFile synpress.config.ts"
```

Comment:
Somehow getting this error and unable to run tests..

<img width="517" alt="image" src="https://github.com/OffchainLabs/retryable-tx-panel/assets/7558499/092f7175-6f20-4c8b-ba5d-03d806aabf22">